atlasdb-api/src/test/java/com/palantir/atlasdb/metrics/MetricPublicationArbiterTest.java

```diff
@@ -104,6 +99,20 @@ public void deduplicatesFilters() {
         assertThat(filters.get(METRIC_NAME_1)).hasSize(2);
     }
 
+    private static MetricPublicationFilter trueFilter() {
```

Comment:
Had to do this because of the `DistinctVarargsChecker` error prone check.

https://app.circleci.com/pipelines/github/palantir/atlasdb/6110/workflows/ed606313-5f90-4757-b80e-0069c1db86fb/jobs/27874

Reply:
ah okay. Yeah this makes sense (also that's a pretty cool error-prone check!)